pipeline/terraform/stack/variables.tf

```diff
@@ -9,7 +9,7 @@ variable "min_capacity" {
 
 variable "max_capacity" {
   type        = number
-  default     = 10
+  default     = 12
```

Comment:
Moar power.

This number is chosen somewhat conservatively to avoid overwhelming the ES/ID minter clusters; with the 2×58GB Elasticsearch instances this was fine for the 2022-06-18 index.

This index also put less load on the `images-indexed` index (see https://github.com/wellcomecollection/catalogue-pipeline/pull/2130), which might have been a contributing factor.